### PR TITLE
Contracts: Drafted more thorough approach.

### PR DESCRIPTION
**Pull request by JoshOrndorff**
## Overview
These contracts should start to address performance testing needs.
Follow-up to https://github.com/rchain/rchain/pull/2246

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-2839

### Notes
* I'm not clear when and whether it is useful to separate the setup and execution phases as is done with `wide.rho`.
* `test_one_comm.rho` and `test_ten_comms.rho` are superseded by the work in `bruteForce`.

### CODE REVIEW

**Comment by JoshOrndorff** [comment]
@lukasz-golebiewski I can't seem to assign reviewers on here. Are you the right person?

File: contracts/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Performance Testing Contracts
+=============================
+
+The rholang contracts in this directory are used to test the performance of the RChain platform under various load profiles. The profile of each contract is described here briefly. It is useful to predict the expected number of comm events manually so that when performance results are surprising data can be compared with expectations in a meaningful.
```
Comment (lukasz-golebiewski) [inline contracts/README.md]: The last sentence finishes abruptly

File: contracts/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+`n` linear sends parred with `n` single receives all over the same channel.
+
+### `persistentListen.rho`
+`n` linear sends parred with a single persistent listen on the same public channel.
+
+### `persistentSend.rho`
+A single persistent send parred with `n` linear listens on the same public channel.
+
+### `contract.rho`
+`n` linear sends parred with a single `contract` on the same public channel. Performance should be identical to `persistentListen.rho` because `contract` is just syntactic suger. But it's always best to test.
```
Comment (lukasz-golebiewski) [inline contracts/README.md]: *sugar

**Review by lukasz-golebiewski — APPROVED** [review]
This looks really nice! 
In the near future we might want to translate makeRholang.js to scala so that it can be easily plugged into gatling

**Review by dzajkowski — APPROVED** [review]

**Pull request merged**